src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/ExtractFeaturesEngine.java

```diff
@@ -201,15 +201,15 @@ static Map<String, String>  createQueryLabels(List<String> labelStringList) {
 
     private void createVQSRInputFromTableResult(final GATKAvroReader avroReader) {
         final org.apache.avro.Schema schema = avroReader.getSchema();
-        if ( schema.getField(LOCATION_FIELD_NAME) == null ) {
-            throw new UserException("Records must contain a location column");
+        if (( schema.getField(LOCATION_FIELD_NAME) == null ) || ( schema.getField(REF_ALLELE_FIELD_NAME) == null) || ( schema.getField("allele") == null)) {
```

Comment:
before we get things automated, this might benefit from a dab of manual formatting

Reply:
Done